### DISCUSSION TITLE
Region dropdown index is not according to database

### DISCUSSION THREAD

**Discussion by wclansar** [breezefront/community#43] [1. Help]
Hello There,

In my account section i can see region drop down index is not according to database index.

 
![region dropdown](https://user-images.githubusercontent.com/10025094/210326399-55a36a54-a84e-4acb-a227-c4b8477902ed.png)

![region db data](https://user-images.githubusercontent.com/10025094/210326478-74101fea-96b6-4492-98b0-fd1aa7a7271a.png)

Thanks

**Comment by vovayatsyuk** [comment]
Thanks for reporting. This issue was fixed and will be included in the next release (this or next week).

https://github.com/breezefront/module-breeze/commit/af850135dcda04fd9a7fb6e88cd6056fdcd1ced8
https://github.com/breezefront/module-breeze/commit/9e994e3b0e5d1c267d6172b327a7322cc17511d1
